stylesheets/components/cards.css

```diff
@@ -121,8 +121,7 @@
   height: 11rem;
 }
 .card-content-top {
-  margin-left: 1rem;
```

Comment:
still you have modify m-left and right too , refhttps://www.w3schools.com/css/tryit.asp?filename=trycss_margin_shorthand_4val

Reply:
<img width="227" alt="image" src="https://user-images.githubusercontent.com/86543154/152649352-24520368-e01b-442a-8d61-433f2a58fa69.png">
I genuinely don't know why are you still able to see my old code, I did refactor this long back😥